internal/server/singleprocess/service_instance.go

```diff
@@ -2,10 +2,13 @@ package singleprocess
 
 import (
 	"context"
+	"time"
 
 	"google.golang.org/grpc/codes"
 	"google.golang.org/grpc/status"
 
+	"github.com/hashicorp/go-memdb"
```

Comment:
Nitpick: there's a newline above this and prob shouldn't be. Our usual import styling is:

```go
import (
  STDLIB

  EXTERNAL

  INTERNAL
)
```